e2e/cypress/integration/links.spec.ts

```diff
@@ -0,0 +1,51 @@
+/// <reference types="cypress" />
+
+import TestContext from "../support/TestContext"
+const path = require('path')
+
+let ctx;
+let proc;
+
+let LINK = 'https://voxel51.com/'
+describe('links', () => {
+    before(async () => {
+      const FIXTURES = path.join(__dirname, '..', 'fixtures')
+      const IMG_PATH = path.join(FIXTURES, 'sample_img.jpeg')
+      ctx = new TestContext(cy)
+      proc = await ctx.py(`
+        import fiftyone as fo
+
+        sample = fo.Sample(
+            filepath="${IMG_PATH}",
+            link="${LINK}",
+        )
+        
+        dataset = fo.Dataset()
+        dataset.add_sample(sample)
+        
+        session = fo.launch_app(dataset)
```

Comment:
The below will prevent browser opens.

```suggestion
        session = fo.launch_app(dataset, remote=True)
```